documentation/airport-format.md

```diff
@@ -1,309 +1,228 @@
----
-title: Airport format
----
-[back to index](index.html)
+# Airport Format
 
-# Airport format
+[NOTICE](## NOTICE)
+[Template](## Template)
+[Comments](## Comments)
+    [Property Descriptions](### Property Descriptions)
+        [Latitude, Longitude, Elevation](#### Latitude, Longitude, Elevation)
+        [ICAO and IATA codes](#### ICAO and IATA codes)
+        [#### `spawnPatterns` Build Process](#### `spawnPatterns` Build Process)
```

Comment:
you should make this Table of Contents into an ordered list. https://guides.github.com/features/mastering-markdown/ _see the `lists` tab under Examples_